core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -566,6 +570,33 @@ private FunctionResolver unix_timestamp() {
     );
   }
 
+  /**
+   * UTC_DATE(). return the current UTC Date in format yyyy-MM-dd
+   */
+  private DefaultFunctionResolver utc_date() {
+    return define(BuiltinFunctionName.UTC_DATE.getName(),
+        implWithProperties(functionProperties
+            -> DateTimeFunction.exprUtcDate(functionProperties), DATE));
+  }
+
+  /**
+   * UTC_TIME(). return the current UTC Time in format HH:mm:ss
+   */
+  private DefaultFunctionResolver utc_time() {
+    return define(BuiltinFunctionName.UTC_TIME.getName(),
+        implWithProperties(functionProperties
+            -> DateTimeFunction.exprUtcTime(functionProperties), TIME));
+  }
+
+  /**
+   * UTC_TIMESTAMP(). return the current UTC TimeStamp in format yyyy-MM-dd HH:mm:ss
+   */
+  private DefaultFunctionResolver utc_timestamp() {
+    return define(BuiltinFunctionName.UTC_TIMESTAMP.getName(),
+        implWithProperties(functionProperties
+            -> DateTimeFunction.exprUtcTimeStamp(functionProperties), DATETIME));
```

Comment:
```suggestion
            ->exprUtcTimeStamp(functionProperties), DATETIME));
```

[Doesn't](https://github.com/Bit-Quill/opensearch-project-sql/pull/187/files#r1044940403) this comment apply here as well?